### PR TITLE
Sanity check the build flag

### PR DESCRIPTION
**Pull request by Kirkkt**
Sanity check the build flag

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,7 +25,7 @@ ONPAGE_TEST=$(subst ",\",$(shell perl -pe 'BEGIN{$$sub="../dist/build.js"};s\#SC
 
 .PHONY: clean
 
-all: dist/build.min.js dist/build.js README.md TITANIUM_GUIDE.md WEB_GUIDE.md example.html test/branch-deps.js dist/build.min.js.gz dist/build.min.js.gz test/integration-test.html
+all: dist/build.min.js dist/build.js README.md TITANIUM_GUIDE.md WEB_GUIDE.md example.html test/branch-deps.js test/integration-test.html
```
Comment (Kirkkt) [inline Makefile]: unrelated change: let's not bother with .gz unless we are making a release

File: package.json [hunk 2]
```diff
@@ -27,8 +27,6 @@
 		"grunt-saucelabs": "~8.6.1",
 		"jsdoc": "3.2.2",
 		"jsdox": "0.4.9",
-		"karma": "^0.12.37",
```
Comment (Kirkkt) [inline package.json]: we are not using karma

**Comment by Kirkkt** [comment]
@dmitrig01 

A handful of miscellaneous things. Please see inline comments for details.

File: src/0_config.js [hunk 3]
```diff
@@ -15,3 +15,10 @@ var CORDOVA_BUILD = false; // jshint unused:false
 
 /** @define {boolean} */
 var TITANIUM_BUILD = false; // jshint unused:false
+
+var isApp = (document.URL.indexOf('http://') === -1) && (document.URL.indexOf('https://') === -1);
```
Comment (Kirkkt) [inline src/0_config.js]: Sanity check the build flag

Comment (dmitrig01) [inline src/0_config.js]: looks great. 

1) i might do an alert, although
2) i would rather check window.cordova instead - this would prevent anyone from running it outside the context of http

Comment (dmitrig01) [inline src/0_config.js]: we should also add these propeties to the exerns file

Comment (dmitrig01) [inline src/0_config.js]: *externs

**Comment by Kirkkt** [comment]
Not sure if this is what you had in mind, PTAL @dmitrig01

**Comment by dmitrig01** [comment]
:+1:
